plenum/server/replica.py

```diff
@@ -919,12 +919,14 @@ def _process_valid_preprepare(self, pre_prepare, sender):
         if why_not_applied is not None:
             return why_not_applied
         self.addToPrePrepares(pre_prepare)
+        
         if self.isMaster:
             # TODO: can pre_state_root be used here instead?
             state_root = self.stateRootHash(pre_prepare.ledgerId, to_str=False)
             self.node.onBatchCreated(pre_prepare.ledgerId, state_root)
-        # BLS multi-sig:
-        self._bls_bft_replica.process_pre_prepare(pre_prepare, sender)
+            # BLS multi-sig:
+            self._bls_bft_replica.process_pre_prepare(pre_prepare, sender)
+
         self.logger.trace("{} saved shared multi signature for "
```

Comment:
The log should also be called for master only...

Reply:
Ok